assets/common/js/models/Dictionary.js

```diff
@@ -370116,4 +370143,24 @@ function Dictionary() {
 Dictionary.prototype.getRandomWord = function() {
 	let index = Math.floor(Math.random() * this.words.length);
 	return this.words[index];
-}
\ No newline at end of file
+}
+
+/**
+ * Function that returns a random word from the subDictonary that corresponds to the current difficulty.
+ * 
+ * @return {string} A random word in all lowercase
+ */
+Dictionary.prototype.getRandomWordForDifficulty = function(gameDifficulty) {
```

Comment:
This seems to receive the entire gameDifficulty as an object from what I can tell.
Since the dictionary object already maintains an instance of gameDifficulty for diffficult parameters references, could we refactor this to accept just a number value for this function?

Reply:
For now, in my Bonus Word story, I got around this problem by locally declaring a temporary gameDifficulty object and initializing it with incremented difficulty level and passing that object to the function.

Reply:
Ahhh yes sure!! No problem
I'm on it